pcsx2/CDVD/CDVD.cpp

```diff
@@ -687,6 +868,36 @@ void cdvdReset()
 	cdvd.RTC.day = (u8)curtime.GetDay(wxDateTime::GMT9);
 	cdvd.RTC.month = (u8)curtime.GetMonth(wxDateTime::GMT9) + 1; // WX returns Jan as "0"
 	cdvd.RTC.year = (u8)(curtime.GetYear(wxDateTime::GMT9) - 2000);
+
+	{
+		FILE *f = fopen("eks.bin", "rb");
```

Comment:
Hey, great to see this PR! We are working heavily on the GS right now so it might take a while for us to review/merge that.

Related to that snippet, you probably shouldn't be trying to read files as hardcoded as this, might want to use our ini settings class to set the path/ put it in the bios lookup system?

I'll wait for others to comment on the rest of the file as I'm working on another PR right now, but I assume this adds support for, in theory, official memcards and arcade systems? The protocol seems to be fully implemented, are there any pitfalls or things we should know since you left the description of your PR empty?

Reply:
Arcade (Namco System 246/256) has some extra security that fails on PCSX2. It initializes the memory card, opens boot.bin, but doesn't try to decrypt. Fails before that. If somebody finds out why this happens it likely means that arcade will work.